static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -100,17 +99,6 @@ const initialiseClickstream = () => {
 
 const loadAnalytics = () => {
     interactionTracking.init();
-    if (config.get('switches.ophan')) {
-        if (config.get('switches.scrollDepth')) {
```

Comment:
Is it useful to keep this switch ready for DCR or do we want to kill that now too?

Reply:
What the point of a switch that’s not hooked into anything? Let’s remove for now.